lib/Neo4j/Types/DateTime.pod

```diff
@@ -108,21 +108,22 @@ C<LOCAL DATETIME>, C<ZONED DATETIME>.
  $tz_name = $dt->tz_name;
 
 Return the name of the time zone to be used for display purposes.
+The name refers to the IANA Olson time zone database.
 For example, the time zone name for California would be
 C<America/Los_Angeles>.
 For C<DATE> values and C<LOCAL *> values, return C<undef>.
 
-To define a display time zone, it is sufficient for either the
-time zone name or a numerical time offset to be available.
-If this method returns C<undef>, you need to use C<tz_offset()>
-instead.
+For a display time zone defined by standard offset only, this
+method will return a zone name in the C<Etc> area. Note that
+its signs are intentionally inverted to conform with POSIX.
```

Comment:
This language isn’t clear: Is it inverted from the numeric offset, or from the Etc zones as defined in Olson’s?